node-installer/node-installer.go

```diff
@@ -199,19 +199,23 @@ func patchContainerdConfig(runtimeName, basePath, configPath string, platform pl
 		existing = constants.ContainerdBaseConfig()
 	}
 
+	snapshotterName := "tardev"
```

Comment:
In the function below you use `"no-snapshotter"` when no snapshotter is needed? Should we also use this here?